### PR TITLE
Phase2-hgx139 New geometry from Shuichi et al.

### PR DESCRIPTION
**Pull request by bsunanda**
Changes the inner part with some support structure

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import FWCore.ParameterSet.Config as cms
```
Comment (ianna) [inline Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py]: @bsunanda - this file should be generated from 2023 dictionary

**Review by ianna — CHANGES_REQUESTED** [review]

File: Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py (hunk 1 above)
Comment (bsunanda) [reply Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py]: I made this using 2023 dictionary in a private way. I discussed with Kevin and he does not want this to be made an official workflow to which I completely agreed. So this cfi is to do some non-official studies. I can rename the file if this sounds like an official name.

Comment (ianna) [reply Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py]: please, do rename it to avoid confusion.

Comment (ianna) [reply Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py]: on the second thought, it has to be generated from a dictionary even if it's not result in a different workflow. We are going this way to all geometry scenarios in any case.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@ianna Can you please take a look yet again? I think I have done what you asked for

File: Configuration/Geometry/README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Calorimeters:
 * C3: HGCal (v8) + Phase2 HCAL and EB
 * C4: HGCal (v9) + Phase2 HCAL and EB
 * C5: HGCal (v9 without virtual wafers) + Phase2 HCAL and EB
+* C4: HGCal (v9a with inner support structure) + Phase2 HCAL and EB
```
Comment (kpedro88) [inline Configuration/Geometry/README.md]: C4 -> C6
but actually C6 -> C7 (C6 is used for HFNose in #24200)

**Comment by bsunanda** [comment]
There is a big conflict with PR #24200 - will submit a new PR once that PR is merged. Kindly merge #24200 at the earliest

**Pull request closed without merging**